src/frontend/App/hooks/felles/useFormState.ts

```diff
@@ -17,7 +17,9 @@ export type FormHook<T extends Record<string, any>> = {
 };
 
 export type FormErrors<T extends Record<string, any | undefined>> = {
-    [P in keyof T]: T[P] extends string | undefined ? string | undefined : FormErrors<T[P]>;
+    [P in keyof T]: T[P] extends string | number | undefined
```

Comment:
Denne ble endret for å kunne legge inn tekst på felter som var av typen number, sånn att number er lagt inn her